src/components/AnnotationsPane.jsx

```diff
@@ -103,16 +104,50 @@ class AnnotationsPane extends React.Component {
   /*  Renders all the annotations that the user has completed.
       WARNING: Not to be confused with annotations from other users!
    */
-  renderAnnotations() {
+  renderUserAnnotations() {
     if (!this.props.annotations) return null;
+    return this.renderAnnotations(this.props.annotations);
+  }
+
+  renderPreviousAnnotations() {
+    if (!this.props.previousAnnotations) return null;
+    return this.renderAnnotations(this.props.previousAnnotations, true);
+  }
+
+  determineGreenLine(annotation, index) {
+    let greenLine = false;
+
+    if (this.props.selectedAnnotation && index === this.props.selectedAnnotationIndex) {
+      if (annotation.previousAnnotation && this.props.selectedAnnotation.previousAnnotation) {
+        greenLine = true;
+      } else if (!annotation.previousAnnotation && !this.props.selectedAnnotation.previousAnnotation) {
+        greenLine = true;
+      }
+    }
+    return greenLine;
+  }
+
+  renderAnnotations(annotations, previousAnnotations = false) {
+    if (!annotations) return null;
     if (!this.props.showPreviousMarks) return null;
 
     const annotationPrefix = 'ANNOTATION_';
 
-    return this.props.annotations.map((annotation, indexOfAnnotation) => {
+    return annotations.map((annotation, index) => {
+      if (annotation.hasCollaborated === true) { return null; }
```

Comment:
I decided to add the `hasCollaborated` tag on previous annotations to toggle to true once a user has selected and saved a text annotation from the dropdown. This is because, once a user interacts with a previous annotation from the dropdown and saves text, that becomes a **new** user annotation. 

This line would return `null` if a previous annotation has already been "collaborated with" so the renderer doesn't attempt to display a user annotation and previous annotation on the exact same line.